_events/baectf.md

```diff
@@ -1,73 +0,0 @@
----
-name: "BAE Systems Applied Intelligence – Capture the Flag 2019"
-is-public: true
```

Comment:
Please can we just flip this bool rather than deleting the whole file?

Reply:
The event will still be on the repo, and will be accessible via /baectf (its slug) - I've spoken with Dulcie, agreed that it's probably best we remove it.